shell/pom.xml

```diff
@@ -82,8 +86,8 @@
       <version>${project.version}</version>
     </dependency>
     <dependency>
-        <groupId>org.apache.commons</groupId>
-        <artifactId>commons-compress</artifactId>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-compress</artifactId>
```

Comment:
please revert if there is no change here

Reply:
See comment above